### PR TITLE
rename runtime to cells to match aurae

### PR DESCRIPTION
**Pull request by dmah42**

### CODE REVIEW

File: cmd/cells/cells.go [hunk 1]
```diff
@@ -35,10 +35,10 @@ import (
 	"io"
```
Comment (slamp) [inline cmd/cells/cells.go:35]: line 31: Shouldn't you also rename 'package runtime' to 'package cells'?

Comment (dmah42) [reply cmd/cells/cells.go:35]: yes :)

**Review by slamp — APPROVED** [review]
lgfm

**Review by slamp — APPROVED** [review]
lgfm

**Pull request merged**